friktion/friktion/friktion_anchor/instructions/__init__.py

```diff
@@ -1,8 +1,6 @@
-# flake8: noqa
-
-from .cancel import CancelAccounts, cancel
-from .claim import ClaimAccounts, claim
-from .create import CreateAccounts, CreateArgs, create
-from .exec import ExecAccounts, exec
-from .exec_msg import ExecMsgAccounts, ExecMsgArgs, exec_msg
-from .set_counterparty import SetCounterpartyAccounts, set_counterparty
+from .create import create, CreateArgs, CreateAccounts
```

Comment:
We should not use relative imports.

Since we would be waiting for the other review, if we could fix the ones we have.

In this case you would import from `friktion.friktion_anchor.instructions.create`, I believe.

Reply:
We could also change `friktion_anchor` to `anchor` by using the absolute imports.

Reply:
hi! everything in `friktion_anchor` and `inertia_anchor` is an automatically generated client based on our solana IDL (equivalent to ABI in EVM). When an update occurs I would run the client generation and the edits we make to those directories would be erased, so I think it may not make sense to make any meaningful changes to that section of the code.